package.json

```diff
@@ -95,7 +95,9 @@
     },
     "overrides": [
       {
-        "files": ["build/**/*"],
+        "files": [
+          "build/**/*"
+        ],
```

Comment:
NPM keeps formatting the file like this when the dependencies change, so I included it here as well.